packages/vite-plugin-svelte-inspector/package.json

```diff
@@ -52,5 +52,8 @@
     "@types/debug": "^4.1.12",
     "svelte": "^5.0.0-next.210",
     "vite": "^5.3.5"
+  },
+  "publishConfig": {
+    "provenance": true
```

Comment:
is there a reason to add this to multiple package.json files instead of setting it in the workspace .npmrc? 

https://docs.npmjs.com/generating-provenance-statements#using-third-party-package-publishing-tools